Bug: "Error: The terraform-provider-crowdstrike_v0.0.6x plugin crashed!"

I've reproduced both locally and on CI runners (fresh terraform container image for each job). Sometimes the Error message presented (with red font) by Terraform changes, but the common theme is that nothing else about this project, which worked just before I updated the provider (via `terraform init -upgrade`), with provider `v0.0.39`, now seems consistently broken.

The main purpose of the repo is to configure module `registry.terraform.io/CrowdStrike/cloud-registration/aws` v`0.4.1` for each of our AWS accounts. When it runs with provider `v0.0.39` (with committed `.terraform.lock.hcl` file, in the main branch), I see: `No changes. Your infrastructure matches the configuration.`

But after running `terraform init -upgrade` and pushing the updated `.terraform.lock.hcl` file, it breaks/fails consistently. 

## Sample error messages:
local, with providers `v0.0.61` and `v0.0.60`
```shell
Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Request cancelled
│ 
│ The plugin6.(*GRPCProvider).ReadResource request was cancelled.
```
```shell
crowdstrike_cloud_aws_account.this: Refreshing state...

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The
│ plugin logs may contain more details.
╵
Releasing state lock. This may take a few moments...

Stack trace from the terraform-provider-crowdstrike_v0.0.60 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
```

ci, with provider v0.0.62
```shell
Planning failed. Terraform encountered an error while generating this plan.
╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more
│ details.
```

## Terraform and plugin versions:

ci runner (alpine:3.22):
```shell
$ terraform version
Terraform v1.14.5
on linux_arm64
```
local (Ubuntu 24):
```shell
tf version
Terraform v1.12.2
on linux_amd64

+ provider registry.terraform.io/crowdstrike/crowdstrike v0.0.62
+ provider registry.terraform.io/hashicorp/aws v5.100.0
```
I also tried upgrading the AWS provider to hashicorp/aws v6.33.0, but that also fails with similar errors

--- a/internal/fcs/cloud_aws_account.go
+++ b/internal/fcs/cloud_aws_account.go
@@ -1023,7 +1023,6 @@ func (m *cloudAWSAccountModel) wrap(ctx context.Context, cloudAccount *models.Do
 		m.IsOrgManagementAccount = types.BoolValue(false)
 	}
 
-	// Set computed values from cloudAccount
 	m.ExternalID = types.StringValue(cloudAccount.ResourceMetadata.ExternalID)
 	m.IntermediateRoleArn = types.StringValue(cloudAccount.ResourceMetadata.IntermediateRoleArn)
 	m.IamRoleArn = types.StringValue(cloudAccount.ResourceMetadata.IamRoleArn)
@@ -1061,7 +1060,9 @@ func (m *cloudAWSAccountModel) wrap(ctx context.Context, cloudAccount *models.Do
 	}
 	m.DspmRoleName = types.StringValue(dspmRoleName)
 	m.DspmRoleArn = types.StringValue(dspmRoleArn)
-	m.DSPM.RoleName = types.StringValue(dspmRoleName)
+	if m.DSPM != nil {
+		m.DSPM.RoleName = types.StringValue(dspmRoleName)
+	}
 
 	var vulnScanningRoleArn string
 	vulnScanningRoleNameFromSettings := settings.VulnerabilityScanningRoleName.ValueString()
@@ -1077,11 +1078,15 @@ func (m *cloudAWSAccountModel) wrap(ctx context.Context, cloudAccount *models.Do
 	}
 	m.VulnerabilityScanningRoleName = types.StringValue(vulnScanningRoleName)
 	m.VulnerabilityScanningRoleArn = types.StringValue(vulnScanningRoleArn)
-	m.VulnerabilityScanning.RoleName = types.StringValue(vulnScanningRoleName)
+	if m.VulnerabilityScanning != nil {
+		m.VulnerabilityScanning.RoleName = types.StringValue(vulnScanningRoleName)
+	}
 
 	agentlessScanningRoleName := dspmRoleName
-	if !m.DSPM.Enabled.ValueBool() && m.VulnerabilityScanning.Enabled.ValueBool() {
-		agentlessScanningRoleName = vulnScanningRoleName
+	if m.DSPM != nil && m.VulnerabilityScanning != nil {
+		if !m.DSPM.Enabled.ValueBool() && m.VulnerabilityScanning.Enabled.ValueBool() {
+			agentlessScanningRoleName = vulnScanningRoleName
+		}
 	}
 	m.AgentlessScanningRoleName = types.StringValue(agentlessScanningRoleName)
 

